docs/core/track_fitting.md

```diff
@@ -1,5 +1,36 @@
 # Track Fitting
 
+The track fitting algorithms estimate the track parameters.
+It is part of the pattern recognition/track  reconstruction/tracking.
+We can run the track fitting algorithms, after we allocated all hits to single tracks with the help of a track finding algorithm.
+It is not necessary, that all points of a track are present.
+
+Currently we have implementations for three different fitters:
+* Kalman Filter
+* GSF
+* Global Chi-Square Fitter (GX2F) [wip]
+Even though all of them are least-squares fits, the concepts are quite different.
+Therefore, we should not expect identical results from all of them.
+
+(kf_core)=
+## Kalman Filter (KF) [wip]
+
+:::{note}
+This is a stub!
+:::
+
+The Kalman Filter is an iterative fitter.
+It successively combines measurements to obtain an estimate of the track parameters.
+The KF needs an estimate as a starting point. The procedure alternates between two methods:
+1. Extrapolate the current state to the next surface.
+2. Update the extrapolation using the measurement of the new surface.[^billoir]
+The meaning of "this surface" and "the next surface" changes with the context.
+There are three different interpretations for this.
+The KF can give us those three interpretations as sets of track parameters:
+* predicted: Uses "older" data (i.e. from the last surface) to make the prediction.
```

Comment:
I would be more precise with "older" data. They are not the exact data from the last surface, but the extrapolation from the last surface without taking into account the measurment on the current surface. 
So the reader should not think that the predicted data are the acutal parameters at the last surface.